docs/plugins.yml

```diff
@@ -0,0 +1,78 @@
+---
+#####################
+# We're excited you wrote plugin! Here's how you share it with others:
+#
+# 1. Add a newline anywhere in the list below
+# 2. List your plugin following the <user>/<repo> format
```

Comment:
Do we do some sort of processing on the repo name (i.e. looking for snap-plugin-{collector,processor,publisher}-x in order to put it into the right section? What happens if I add a plugin like `ircody/super-cool-plugin`?

If it requires the repo to be named something specifically maybe it's worth calling out.